scripts/mirror.js

```diff
@@ -282,71 +244,55 @@ const bumpChromeAndroid = (originalData, sourceData) => {
 /**
  * @param {SupportStatement} originalData
  * @param {SupportStatement} sourceData
```

Comment:
I'm not exactly sure what these contain. Is `sourceData` the _latest_ support statement of the browser from which we're mirroring?